### ISSUE TITLE
Change tag component border colour to subtler colours

### ISSUE THREAD

**Issue by hazalarpalikli** [alphagov/govuk-frontend#6542]
## What
We have added a border to the tag component to solve the legibility issue with the text and the background colour as a result of brand colour changes. This caused the component to look more like a button. 

To make it look less like a button, we are suggesting of using a subtler colour for the border rather than matching the colour with the text colour for code consistency. 

(Subtler option is what we trialled intially, it can also be found in the previous PRs) 

## Why
So that we solve the issue of the component looking like a button and confusing users. 

## Who needs to work on this
Designer, Developer 

## Who needs to review this
Team

## Done when

- [ ] Colours are chosen for the component
- [ ] A pr is created for the changes 
- [ ] Changes are shared with the team

**Comment by 36degrees** [comment]
Moving this to the govuk-frontend repo as it's about making changes to the design of a component.

**Comment by kellylee-gds** [comment]
Added a "blocked by" marker as we've agreed in Xmas small stories week, to work on [#1974](https://github.com/alphagov/govuk-design-system/issues/1974) before picking this up.

**Comment by hazalarpalikli** [comment]
I have done some exploration in this work:

1-  Prototype 1 using losanges : https://github.com/alphagov/govuk-frontend/pull/6552

2- Using subtler colours:  
With the subtler colour, some of the text is looking less legible, see yellow tag on the image 

<img width="980" height="673" alt="screenshot showing ds team tag list" src="https://github.com/user-attachments/assets/98dd57c5-e82d-4e40-87b6-24081a945d01" />

**Comment by lewisdarlow-gds** [comment]
<img width="1301" height="841" alt="A task list from Task A to Task L in alphabetical order with rectangle tags which the tags have a border, text and are different colours" src="https://github.com/user-attachments/assets/43ea82ab-9cb6-49e5-80be-d9367e088d49" />

I also did a bit of an experiment which was looking at removing the fill colour and just having 3px border to make the tag feel unique compared to our button and to take less visual presence. 

The yellow especially doesn't have enough contrast with the text against white. Looking at the version you've done @hazalarpalikli which if the text is yellow 50% shade and the background is yellow 80% tint according to whocanuse.com this still doesn't meet the contrast ratio just wanted to flag it here.

**Comment by mia-allers-gds** [comment]
Spicy Q but if the yellow is tripping us up then maybe yellow is not appropriate for tags?

**Comment by hazalarpalikli** [comment]
I am doing some experiments to remove the border and maybe reverse the colours similar to older versions of the tag component. 

Some drawback with his:
- Some of the text see orange and purple, however this could be solved with using lighter tones. 
- Colours red/orange , turquoise/green are looking very similar to one another 

I haven't done the accessibility checks for these (not yet) 

<img width="988" height="622" alt="screenshot showing task list page" src="https://github.com/user-attachments/assets/3ae82f5a-d0a6-4f13-900d-e0827e182a58" /> 

For the phase banner i am suggesting these colours, first image is not reflecting the colour changes 

<img width="126" height="56" alt="screenshot showing phase banner tag" src="https://github.com/user-attachments/assets/5fb2b009-08b5-44d7-950c-c6c431f18bb4" />

**Comment by hazalarpalikli** [comment]
Header with phase banner: 

<img width="616" height="116" alt="screenshot showing header with phase banner" src="https://github.com/user-attachments/assets/86636421-bfbd-438b-b67d-253473214a1d" />

**Comment by hazalarpalikli** [comment]
Some proposed changes for the dark background exploration and tests with APCA

| Tag | Colour  | APCA Lc Value  | 
| -------- | ------- | ------- |
| <img width="52" height="32" alt="grey tag" src="https://github.com/user-attachments/assets/76db3324-bce5-4c80-9e60-ce417b7023bb" /> |  text-colour: #F3F3F3 (black-tint-95)  background-colour: #484949 (black-tint-25) |  -87.3 |
| <img width="60" height="30" alt="blue tag" src="https://github.com/user-attachments/assets/94fe6820-f86a-4f6b-942c-b58200f8dc56" />| text-colour: #F4F8FB (blue-tint-95)  background-colour: #1D70B8 (blue)| -75.5 |
| <img width="347" height="106" alt="light blue tag" src="https://github.com/user-attachments/assets/ff2e2e33-4abc-4d35-ab2c-f833fab7e27f" />| text-colour: #0F385C (blue-shade-50) background-colour: #8EB8DC (blue-tint-50) | 54.3 |
| <img width="210" height="71" alt="teal tag" src="https://github.com/user-attachments/assets/6b1a33f1-511e-4776-bdde-afbc1375aa01" /> |  text-colour:#F3F9F9(teal-tint-95) background-colour:  #158187(teal) | -72.5 |
| <img width="163" height="86" alt="green tag" src="https://github.com/user-attachments/assets/fb249bd4-2633-4b13-98fc-d6cd5e56a0f1" /> | text-colour: #F3F9F7 green-tint-80 background-colour:  #11875A green-shade-50 | -71.5|
| <img width="191" height="91" alt="purple tag" src="https://github.com/user-attachments/assets/8a996af1-1128-4718-a714-b0d76342842f" /> | text-colour: purple-tint-95 #F6F5FA, background-colour: purple #54319F| -88.5 |
| <img width="145" height="92" alt="magenta tag" src="https://github.com/user-attachments/assets/5bb80d7f-a641-4fa1-a184-8f734a403fb4" /> | text-colour: tint-95 #FCF5F8 background-colour: magenta #CA357C | -72.5 |

**Comment by lewisdarlow-gds** [comment]
I have two experiments:  

## Experiment 1:  
- changing the tag border and text to shade-25
- a white background fill on the tags 
- adding a corner-radius to change it from a rectangle to a pill   

I’ve created a pull request #6601 and you can see it [on the review app](https://govuk-frontend-pr-6601.herokuapp.com/), which I've looked at in context of the phase banner and task list. 

### Accessibility
I’ve checked it against WCAG and APCA :  the yellow currently fails on both WCAG and APCA.  the orange currently is four points out on passing the APCA this could be solved. I can put the ratings into a document as currently they're in Figma. 

This experiment I think
- improves contrast 
- differentiates the button and tag 

Drawback
- it’s harder to tell the difference between the “blue” and “light blue” tag - which I think we said in a design meeting we aren’t sure why they’re two so similar 


## Experiment 2: 

- changing the tag border and text to shade-25
- a combination of tint-80 and tint-95 background
- adding a corner-radius to change it from a rectangle to a pill   

There is a pull request #6600 and you can see [it on the review app](https://govuk-frontend-pr-6600.herokuapp.com/). 

Drawback
- this is very similar from above and does have similar issues with the yellow and orange 

Questions:
- how do we solve tags on a dark background @mia-allers-gds said we could inverse the colours so it was white on blue.

**Comment by selfthinker** [comment]
> * how do we solve tags on a dark background [@mia-allers-gds](https://github.com/mia-allers-gds) said we could inverse the colours so it was white on blue.

I think the problem with tags on a dark background is worse with experiment 1. With experiment 2 you also have the lighter background colour of the tags which makes it harder to confuse with anything else.

**Comment by lewisdarlow-gds** [comment]
Thanks @selfthinker that's helpful to know. 

I'm not fully sure in what context we use tags on a darker background, but if it was using tags on the gov.uk blue in theory would any of them pass for contrast?

**Comment by selfthinker** [comment]
> I'm not fully sure in what context we use tags on a darker background, but if it was using tags on the gov.uk blue in theory would any of them pass for contrast?

Their contrast would be the same because the colours are the same. The main difference is that the border would "disappear" as it would be blue on blue.

I have tried to find an example of tags used on a darker background in the tags discussion on GitHub but there are none in there. The main context I'm thinking of would be a phase banner on a service home page, like the Design System's home page. But that's not officially supported (I think), but certainly out there.

**Comment by hazalarpalikli** [comment]
We have made a decision on a direction to go forward. All the discussion is on this board: https://app.mural.co/t/govukdelivery7534/m/govukdelivery7534/1768408750304/a51b1e784aeff5ca009dd7f78b7e484aceec5a71 

In summary, we'll:
- Go with the borderless option and improve the contrast on colours, yellow and light blue. 
- We'll run accessibility checks on the colour combinations
- If the checks fail we'll chat about whether or not retiring is an option with the dev team. 


I am closing this ticket now and will create another one for the checks.
